pages/stakeholder-and-user-interviews.md

```diff
@@ -30,3 +30,7 @@ To build consensus about the problem statement and research objectives.
 ## Applied in government research
 
 No PRA implications. The PRA explicitly exempts direct observation and non-standardized conversation, 5 CFR 1320.3(h)3. See the methods for [Recruiting](../recruiting/) and [Privacy](../privacy/) for more tips on taking input from the public.
+
+## Additional resources
+
+ - [&ldquo;Build empathy with stakeholder interviews, part 1: Preparation.&rdquo;](https://18f.gsa.gov/2016/06/20/build-empathy-with-stakeholder-interviews-part-1-preparation/) Andrew Maier.
```

Comment:
It's interesting that this link is surrounded in quotes--is that how other blog posts are formatted in the method cards?  I'm used to the underline applied by the hyperlink being enough, but I also am unfamiliar with this particular repository and its styling/content.